lib/metrics.js

```diff
@@ -84,17 +84,9 @@ function getTotalNumberOfAppointments(scraperResult) {
     let totalNumberAppointments = 0;
     const numberAppointments = (item) => {
         let ret = 0;
-        if (
-            item?.hasAvailability &&
-            item?.availability &&
-            Object.keys(item.availability).length
-        ) {
+        if (item?.availability) {
```

Comment:
do you know why this file changed?

Reply:
I don't. Did I commit a change to metrics.js? I hope not because I haven't ever tinkered with it (I'm not even sure what it's supposed to do). Is it possible it was so artifact of me updating (merge/pull) from upstream, and then ... I have no idea. I'm not very experienced with git.

Reply:
I just checked and we definitely want the metrics.js file to have no diff here - the things that were reverted need to be put back. Once you can do that, @pfurbacher , I can merge!